### ISSUE TITLE
In the same experiment, some epochs are fast, and some epochs are extremely slow. I wonder why

### ISSUE THREAD

**Issue by l2181265** [MIC-DKFZ/nnUNet#1194]
2022-10-17 18:18:55.618151: 
epoch:  249 
2022-10-17 18:22:08.584400: train loss : -0.6699 
2022-10-17 18:23:12.176860: validation loss: -0.6583 
2022-10-17 18:23:12.178313: Average global foreground Dice: [0.8089] 
2022-10-17 18:23:12.178408: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-10-17 18:23:12.802834: lr: 0.006156 
2022-10-17 18:23:12.803058: This epoch took 257.184862 s
 
2022-10-17 18:23:12.803106: 
epoch:  250 
2022-10-17 18:26:39.668533: train loss : -0.6563 
2022-10-17 18:29:07.345066: validation loss: -0.6603 
2022-10-17 18:29:07.346099: Average global foreground Dice: [0.8364] 
2022-10-17 18:29:07.346193: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-10-17 18:29:08.313486: lr: 0.006141 
2022-10-17 18:29:08.313790: This epoch took 355.510636 s
 
2022-10-17 18:29:08.313889: 
epoch:  251 
2022-10-17 18:42:08.124656: train loss : -0.6949 
2022-10-17 18:48:15.905522: validation loss: -0.7118 
2022-10-17 18:48:15.906203: Average global foreground Dice: [0.8314] 
2022-10-17 18:48:15.906278: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-10-17 18:48:16.984188: lr: 0.006125 
2022-10-17 18:48:16.984418: This epoch took 1148.670440 s
 
2022-10-17 18:48:16.984466: 
epoch:  252 
2022-10-17 19:17:02.628314: train loss : -0.6753 
2022-10-17 19:23:40.429017: validation loss: -0.7192 
2022-10-17 19:23:40.429847: Average global foreground Dice: [0.8505] 
2022-10-17 19:23:40.429945: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-10-17 19:23:41.047838: lr: 0.006109 
2022-10-17 19:23:41.048041: This epoch took 2124.063530 s

**Comment by AijiAo-TL** [comment]
Is your dataset 2D?

**Comment by ykirchhoff** [comment]
Hi @l2181265,

this is most likely due to other processes running on your machine consuming either CPU or GPU resources.

Best,
Yannick
